api/src/services/faucet.js

```diff
@@ -26,9 +26,21 @@ module.exports = {
 
       const client = await getClient();
 
+      // Check if SUPERUSER_API_TOKEN is not defined or not a string
+      if (typeof SUPERUSER_API_TOKEN !== 'string') {
+        console.log('Warning: SUPERUSER_API_TOKEN env is not defined or not a string');
```

Comment:
Let's rename `not a string` to `has incorrect format`

Reply:
Fixed.